_posts/2017-07-20-Rust-1.19.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: post
+title: "Announcing Rust 1.19"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.19.0. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed, getting Rust 1.19 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.19.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1190-2017-07-20
+
+### What's in 1.19.0 stable
+
+Rust 1.19.0 has some long-awaited features, but first, a note for our Windows
+users. On Windows, Rust relies on `link.exe` for linking, which you can get
+via the "Microsoft Visual C++ Build Tools." With the recent release of Visual
+Studio 2017, the directory structure for these tools has changed. As such, you
+had to stick with the 2015 tools to use Rust, or use a workaround (such as running
+`vcvars.bat`). In 1.19.0, `rustc` now knows how to find the 2017 tools, and so
+they work without a workaround.
+
+On to new features! Rust 1.19.0 is the first release that supports [`union`s]:
+
+```rust
+union MyUnion {
+    f1: u32,
+    f2: f32,
+}
+```
+
+Unions are kind of like `enum`s, but they are "untagged". Enums have a "tag"
+that stores which variant is the correct one at runtime; unions elide this tag.
+
+That means that reading or writing a union's field is unsafe:
```

Comment:
I don't get why unions eliding the tag means reading or writing a union's field is unsafe exactly... This isn't really the right place to elaborate completely, but I wonder if we could make this sentence just a little bit clearer, something like (my best guess as to the reason here):

"Since we can interpret the data held in the union using the wrong variant and Rust can't check this for us, that means reading or writing a union's field is unsafe:"